src/Chainweb/Pact/Types.hs

```diff
@@ -519,51 +510,23 @@ defaultOnFatalError lf pex t = do
 
 type ModuleInitCache = M.Map BlockHeight ModuleCache
 
+data PactBlockEnv logger tbl = PactBlockEnv
+  { _psServiceEnv :: PactServiceEnv logger tbl
```

Comment:
missing bangpattern, or was this intentional to avoid forcing the service env more than once

Reply:
Nah I can add it with no problem.